extensions/wikia/Chat2/css/ChatWidget.scss

```diff
@@ -26,162 +28,88 @@ body.skin-oasis .ChatMonobookWidget {
-		font-size: 12px;
-		line-height: 21px;
-		margin: 0;
-		padding: 0;
+	.wds-button.wds-is-secondary {
+		border-color: $color-links;
+		color: $color-links;
+		margin-left: 6px;
 	}
 
-	.chat-join {
-		position: absolute;
-		top: 0;
-		right: 0;
+	.wds-button.wds-is-secondary:hover {
+		border-color: rgba($color-links, .75);
+		color: rgba($color-links, .75);
+		text-decoration: none;
 	}
 
-	.chat-whos-here {
-		@include clearfix;
-		border: 1px solid $color-page-border;
-		margin: 10px 0;
-		padding: 5px;
+	.chat-contents {
 		position: relative;
 
-		.arrow-left,
-		.arrow-right {
-			cursor: pointer;
-			line-height: 34px;
-			position: absolute;
-
-			&.disabled {
-				cursor: auto;
+		&.chat-room-empty {
+			.wds-avatar {
 				opacity: .5;
 			}
 		}
+	}
 
-		.arrow-left {
-			left: 0;
-			padding-left: 10px;
+	.chat-details {
+		@include align-items(center);
+		@include flexbox();
 
-			img {
-				@include arrow(6px, $color-chat-live, 'left');
-			}
-		}
+		.avatars {
+			@include align-items(center);
+			@include flexbox();
+			@include flex(1);
 
-		.arrow-right {
-			right: 0;
-			padding-right: 10px;
+			.wds-avatar-stack {
+				margin: 0;
+				max-width: 120px;
 
-			img {
-				@include arrow(6px, $color-chat-live, 'right');
+				a {
+					height: 32px;
+				}
 			}
-		}
 
-		.carousel-container {
-			height: 34px;
-			left: 21px;
-			overflow: hidden;
-			position: relative;
-			width: 244px;
-			& > div {
-				position: relative;
-			}
-			ul {
-				@include clearfix;
-				list-style: none;
-				margin: 0;
-				position: absolute;
-			}
-			li {
-				float: left;
-				margin: 0 8px 0 0;
-			}
-		}
-		@if ($responsive-or-breakpoints-layout-enabled) {
-			.carousel-container {
-				left: 19px;
-				width: 218px;
+			.more-users-count {
+				font-weight: bold;
+				margin-left: 6px;
 
-				li {
-					margin-right: 12px;
+				&:hover {
+					color: rgba($color-links, .75);
+					text-decoration: none;
 				}
 			}
 		}
+	}
 
-		.avatar {
-			border: 1px solid $color-chat-avatar-border;
-			vertical-align: middle;
-		}
+	.more {
+		font-size: $wds-typescale-size-minus-2;
+		font-weight: normal;
+		margin-top: 16px;
+	}
+
+	h2.chat-headline {
+		border-bottom: 1px solid $color-page-border;
+		height: 30px;
+		margin-bottom: 8px;
 	}
 
-	&.module h3 {
-		font-size: 13px;
-		padding: 0;
+	.chat-join {
+		position: absolute;
+		top: 0;
+		right: 0;
 	}
 }
 
 // Note: a lot of this is borrowed from .WikiaRail .module
 .ChatWidget {
 	@include clearfix;
-	@include box-shadow(0px, 2px, 5px, darken($color-page,12%));
-	background: $color-page;
-	border: 1px solid $color-page-border;
-	clear: both;
-	display: inline-block;
-	@if lightness($color-page) < 50 {
-		@include box-shadow(0px, 2px, 5px, darken($color-page,7%));
-		background: $color-module-background;
-		border: 1px solid $color-page-border;
-	}
-	margin: 0 auto;
-	margin-bottom: 10px;
-	padding: 16px 8px 8px;
+	margin: 0 auto 10px;
 	position: relative;
-	width: 284px;
+	width: 300px;
```

Comment:
I saw this 300px width also in other RR modules - maybe it would be a good idea to have it in a common variable.